CONTRIBUTING.md

```diff
@@ -15,8 +15,8 @@ Generally it's the
 a few additions:
 
 * Any code merged into the Ninja codebase which will be part of the main
-  executable must compile as C++03. You may use C++11 features in a test or an
-  unimportant tool if you guard your code with `#if __cplusplus >= 201103L`.
+  executable must compile as C++11. You may use C++14 features in a test or an
```

Comment:
As the Google C++ Style Guide contains "Consider portability to other environments before using features from C++14 and C++17 in your project." I think that we can remove this bullet point to keep things simple.

Reply:
Sounds good. Done.